### ISSUE TITLE
Split Scheduling Page Into Multiple Pages - Accessibility Review Response

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#18038]
## User Story
As a veteran with accessibility issues, I want the pages I use on VA.gov to be clear and not confusing.

## Expected Behavior
Go from one page to more than one (whatever was decided in the 508 story).

Based on #17766 accessibility feedback.

## Acceptance Criteria
- [ ] Josh Kim gives us a thumbs up

**Comment by kevinstachura** [comment]
The split-page design concept has been put into an Invision prototype: 

https://vsateams.invisionapp.com/share/JHZTPGYBEYV

When this is approved by all parties, I will put the concept into the original HLR design proto. 👍

**Comment by kevinstachura** [comment]
This should be complete as well. Link is here:

https://vsateams.invisionapp.com/share/W5U21BDFTQK#/442114037_HLR-Request-Informal-Conference-2-Who-To-Call
